amulet/level/formats/sponge_schem/format_wrapper.py

```diff
@@ -97,191 +151,117 @@ def _create(
         self._has_lock = True
 
     def open_from(self, f: BinaryIO):
-        sponge_schem = load_nbt(f).compound
-        version_tag = sponge_schem.get("Version")
-        if not isinstance(version_tag, IntTag):
-            raise SpongeSchemReadError("Version key must exist and be an integer.")
-        version = version_tag.py_int
-        if version == 1:
-            raise SpongeSchemReadError(
-                "Sponge Schematic Version 1 is not supported currently."
-            )
-        elif version == 2:
```

Comment:
The removal of this indentation is making this quite difficult to review.
Can you make this `elif 2 <= version <=3:` and move everything back under this.
We can remove it after if we want.

Reply:
Okay, I'll make the corrections and try pushing as soon as possible.